plugins/hadoopsecuritymanager-common/src/azkaban/security/commons/HadoopSecurityManager.java

```diff
@@ -46,6 +47,13 @@ public boolean isHadoopSecurityEnabled()
     return false;
   }
 
+  public void reloginFromKeytab() throws IOException {
+    UserGroupInformation ugi = UserGroupInformation.getLoginUser();
+    if (ugi.isFromKeytab()) {
+      ugi.checkTGTAndReloginFromKeytab();
```

Comment:
Should just directly call `reloginFromKeytab()` because it does all the same checks as `checkTGTAndReloginFromKeytab()`. The `isFromKeytab()` check is also not needed as `reloginFromKeytab()` already does this check. See https://github.com/apache/hadoop/blob/trunk/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java#L1091.